api/routes/collections.js

```diff
@@ -0,0 +1,207 @@
+const express = require('express');
+const router = express.Router();
+const db = require('../db');
+const dbUtils = require('../utils/db.js');
+const apiUtils = require('../utils/api.js');
+
+/* Create a collection */
+router.post('/', (req, res) => {
+  let hostname = req.protocol + '://' + req.headers.host;
+
+  // Validate data received
+  if (!("name" in req.body)) {
+    let err = { "Error": "The request object is missing the name field" };
+    return res.status(400)
+      .set({ "Content-Type": "application/json" })
+      .send(err);
+  }
+
+  // Build query object
+  let query = { 
+    text: 'INSERT INTO "UserCollection"("userID", name, "isPrivate") VALUES($1, $2, $3) RETURNING *', 
+    values: [] 
+  };
+
+  // TODO: Add user ID
+  query.values.push(1);
+  query.values.push(req.body.name);
```

Comment:
Any particular reason why we don't just put these values into the array on line 22 instead of pushing it?

Reply:
I originally did this since I wasn't sure how we'd end up getting the user info. I've made an update to separate out the "placeholder" for the user ID, which hopefully makes it a bit cleaner